crates/protocol/src/batch/bits.rs

```diff
@@ -0,0 +1,224 @@
+//! Module for working with span batch bits.
+
+use crate::SpanBatchError;
+use alloc::{vec, vec::Vec};
+use alloy_rlp::Buf;
+use core::cmp::Ordering;
+
+/// Type for span batch bits.
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+pub struct SpanBatchBits(pub Vec<u8>);
```

Comment:
Since we have `get_bit` and `set_bit`, I don't think having the inner `Vec<u8>` marked `pub` makes sense